### PR TITLE
Add Sortable Behavior to ReorderController validateModel

### PR DESCRIPTION
**Pull request by mjauvin**
required for https://github.com/octobercms/library/pull/519

This adds the Sortable behavior to the required implementation for the ReorderController's model.

### CODE REVIEW

File: modules/backend/behaviors/ReorderController.php [hunk 1]
```diff
@@ -212,17 +212,18 @@ public function reorderGetRecordName($record)
     protected function validateModel()
     {
         $model = $this->controller->reorderGetModel();
-        $modelTraits = class_uses($model);
+        $modelImplements = class_uses($model);
```
Comment (LukeTowers) [inline modules/backend/behaviors/ReorderController.php]: I don't think class_uses() includes behaviors

Comment (mjauvin) [reply modules/backend/behaviors/ReorderController.php]: Dang!

Comment (mjauvin) [reply modules/backend/behaviors/ReorderController.php]: fixed

**Comment by bennothommo** [comment]
Blocked by https://github.com/octobercms/library/pull/519.

File: modules/backend/behaviors/ReorderController.php [hunk 2]
```diff
@@ -214,15 +214,18 @@ protected function validateModel()
         $model = $this->controller->reorderGetModel();
         $modelTraits = class_uses($model);
 
-        if (isset($modelTraits[\October\Rain\Database\Traits\Sortable::class])) {
+        if (
+            isset($modelTraits[\October\Rain\Database\Traits\Sortable::class]) ||
+            $model->isClassExtendedWith('October.Rain.Database.Behaviors.Sortable')
```
Comment (mjauvin) [inline modules/backend/behaviors/ReorderController.php]:
```suggestion
            $model->isClassExtendedWith('October\Rain\Database\Behaviors\Sortable')
```

File: modules/backend/behaviors/ReorderController.php [hunk 3]
```diff
@@ -214,15 +214,18 @@ protected function validateModel()
         $model = $this->controller->reorderGetModel();
         $modelTraits = class_uses($model);
 
-        if (isset($modelTraits[\October\Rain\Database\Traits\Sortable::class])) {
+        if (
+            isset($modelTraits[\October\Rain\Database\Traits\Sortable::class]) ||
+            $model->isClassExtendedWith('October\Rain\Database\Behaviors\Sortable')
```
Comment (mjauvin) [inline modules/backend/behaviors/ReorderController.php]:
```suggestion
            $model->isClassExtendedWith(\October\Rain\Database\Behaviors\Sortable::class)
```

**Pull request merged**